.github/ISSUE_TEMPLATE/task.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Task
+about: Generic work to be done, not directly related to new features
```

Comment:
"**g**eneric"?

Reply:
![image](https://user-images.githubusercontent.com/40166342/156227683-ebbfec96-7286-4279-a10c-c6f1e486a7ae.png)

This is the mask you will see when creating a new issue (after merge to master) by clicking `New issue`. Do you still think lowercasing it would be better?

Reply:
Yes.